charts/argocd-apps/values.yaml

```diff
@@ -30,8 +30,8 @@ applications: []
 #     automated:
 #       prune: false
 #       selfHeal: false
-#   syncOptions:
-#   - CreateNamespace=true
+#     syncOptions:
+#     - CreateNamespace=true
```

Comment:
Ref: https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/
supposed to be `spec. syncPolicy.syncOptions` , not `spec.syncOptions` 😇 
<img width="796" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/f37041cf-7347-4512-893c-8594698f4db6">